linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/dashboard.js

```diff
@@ -11,11 +11,13 @@ $.when(
 ).done(function(overviewStatsRsp, metricsJson) {
   appendOverviewSection();
   var procInfo = ProcInfo();
+  var dashboard = Dashboard();
+
+  var metricsListeners = [procInfo, dashboard];
+  var metricsCollector = MetricsCollector(metricsListeners);
```

Comment:
Oh sure! I just wanted to be clear what they were, so people adding future modules would know explicitly what those were.  Would you like me to update this to not have the var?